### PR TITLE
japi-compliance-check

### PR DESCRIPTION
**Pull request by vdonadze**

### CODE REVIEW

File: .github/workflows/japi-compliance-checker.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        with:
+          distribution: ${{ inputs.javaVendor }}
+          java-version: ${{ inputs.javaVersion }}
+
+      - name: install japi-compliance-checker
+        run: sudo apt-get update && sudo apt-get install -y japi-compliance-checker
+
+      - name: Download target jar
+        run: curl -o targetVersion.jar -L ${{ inputs.targetJarUrl }}
+
```
Comment (OptimumOpium) [inline .github/workflows/japi-compliance-checker.yml:64]: Please add gradle setup before executing it
```
      - name: Setup Gradle
        uses: gradle/gradle-build-action@v2
```
Maybe there is a later version of this action. Could you please check?

**Review by OptimumOpium — CHANGES_REQUESTED** [review]

File: .github/workflows/japi-compliance-checker.yml (hunk 1 above)
Comment (vdonadze) [reply .github/workflows/japi-compliance-checker.yml:64]: Seems like v2 is the latest

File: .github/workflows/japi-compliance-checker.yml [hunk 2]
```diff
@@ -75,6 +84,12 @@ jobs:
           fi
 
       - name: Run japi-compliance-checker
+        if: ${{ inputs.continueOnError }}
+        continue-on-error: true
```
Comment (OptimumOpium) [inline .github/workflows/japi-compliance-checker.yml:88]: I think we should be able to use the parameter directly
```suggestion
        continue-on-error: ${{ inputs.continueOnError }}
```

Could you please try this?

Comment (vdonadze) [reply .github/workflows/japi-compliance-checker.yml]: worked

Comment (OptimumOpium) [reply .github/workflows/japi-compliance-checker.yml]: Cool :+1:

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**